pages/curriculum-workshop-feb-16-2017.jsx

```diff
@@ -0,0 +1,62 @@
+var React = require('react');
+var HeroUnit = require('../components/hero-unit.jsx');
+// use this LinkAnchorSwap component for hyperlinks
+var LinkAnchorSwap = require('../components/link-anchor-swap.jsx');
+var Illustration = require('../components/illustration.jsx');
+
+var CurriculumWorkshop = React.createClass({
+  statics: {
+    pageClassName: 'curriculum-workshop',
+    pageTitle: 'Curriculum Workshops'
+  },
+  render: function () {
+    return (
+        <div className="inner-container call-container">
+          <section className="intro intro-after-banner">
+           <Illustration
+             height={175} width={175}
+             src1x="/img/pages/community/svg/icon-circle-community.svg"
+             alt="icon toolkit">
+               <h1>Mozilla Curriculum Workshop</h1>
+               <h2>Join us on the second Tuesday of each month!</h2>
+           </Illustration>
+          </section>
+
+          <p>
+            Co-hosts Amira Dhalla and Chad Sansing, along with producers Kristina Gorr and Paul Oh, help participants answer the question, <em>"How can I use the web to teach and learn what’s important to me?"</em> Join us as we prototype teaching and learning materials live on-air and think out-loud through the curriculum design process.
+          </p>
+
+          <section className="callout-box past-workshop">
+            <h2>Past Workshop</h2>
+            <p className="date">February 16th - 7am PT/ 10am ET/ 2pm UTC</p>
```

Comment:
@hannahkane it seems like other archived pages are using `,` instead of `/` to separate the times (for example, https://learning.mozilla.org/en-US/community/curriculum-workshop/april-12-2016/ ). For consistency reason can we change `\` into `,`?